presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -339,6 +339,20 @@ as Hive. For example, converting the string ``'foo'`` to a number,
 or converting the string ``'1234'`` to a ``tinyint`` (which has a
 maximum value of ``127``).
 
+SQL Standard Based Authorization
+--------------------------------
+
+Roles and permission management statements are supported by Hive connector
+with some limitations:
+
+* ``CREATE ROLE role WITH ADMIN`` is not supported.
+* Since ``WITH ADMIN`` clause is not supported, only a global admin can execute ``DROP ROLE``.
```

Comment:
Alternatively: The ``admin`` role must be enabled to execute ``CREATE ROLE`` or ``DROP ROLE``.